### DISCUSSION TITLE
Umm... Why is my account restricted?

### DISCUSSION THREAD

**Discussion by CalvinatorIA** [codewars/codewars.com#2659] [Q&A] [answered]
My profile page says "Account restricted for violation of terms of service." I also cannot practice to train anymore. After I complete a training, it just says, ""You have not earned access to this kata's solutions. Solutions are locked for kata ranked far above your rank. Rank up or complete this kata to view the solutions. Why? Did I do something wrong??

**Comment by hobovsky** [comment, marked as answer]
I was not the one who restricted your account, but I think it could have something to do with seemingly unreasonable difficulty/rank votes cast on some beta kata.
I removed the restriction from your account, but please remember that beta votes (rank votes and satisfaction votes) are meant to be actually helpful for reviewers, and they are not there just to collect points.

Thanks.

**Reply by CalvinatorIA** [reply]
Got it! Sorry about encouraging reviewers to provide feedback because of points.
